tests/protractor/helper.js

```diff
@@ -95,11 +88,4 @@ module.exports = {
     var selectedElement = element(by.xpath('//*[text()[normalize-space() =  \' ' + text + '\']]'));
     return selectedElement.isPresent();
   },
-
-  takeScreenshot: function (filename) {
-    browser.takeScreenshot().then(function (png) {
-      writeScreenShot(png, filename);
```

Comment:
What are these screenshots used for?  Do we always need them?  Do we need them on Travis?  Can they be enabled specifically when wanted?